t/cobrand/brent.t

```diff
@@ -125,7 +125,28 @@ my $parks_contact2 = $mech->create_contact_ok(body_id => $brent->id, category =>
 my $parks_contact3 = $mech->create_contact_ok(body_id => $brent->id, category => 'Ponds',
     email => 'ponds@example.org', group => 'Parks and open spaces');
 my $user1 = $mech->create_user_ok('user1@example.org', email_verified => 1, name => 'User 1');
-my $staff_user = $mech->create_user_ok('staff@example.org', from_body => $brent, name => 'Staff User');
+my $role = FixMyStreet::DB->resultset("Role")->create({
+    body => $brent,
+    name => 'Role',
+    permissions => ['moderate', 'user_edit'],
+});
+my $staff_user = $mech->create_user_ok('staff@example.org', from_body => $brent, name => 'Staff User', email_verified => 1);
```

Comment:
Doesn't matter, but email_verified => 1 isn't needed, it's automatically included when manually creating a user with an email address.